src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -662,6 +749,90 @@ public bool KeyEventsEnabled
             }
         }
 
+        /// <summary>
+        /// Background color of web page.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector4 DocumentBackgroundColor
```

Comment:
When I first commented about the name, I though it indicates [document](https://developer.mozilla.org/en-US/docs/Web/API/Document) of the web page literally.
<img width="849" alt="스크린샷 2021-04-09 오후 4 23 55" src="https://user-images.githubusercontent.com/13619032/114144234-11343680-9950-11eb-9e8b-2af2f043485a.png">

Is it correct? The API sets the background color of `document`?

Reply:
after this property is set, background color of ewk view of chromium-efl would be changed.
so it is the same as background color of NUI view.
here this property would be overridden.